src/ResourceManager/SiteRecovery/documentation/upcoming-breaking-changes.md

```diff
@@ -25,4 +25,81 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes
\ No newline at end of file
+# Upcoming Breaking Changes
+
+## Release 5.0.x - month 2018
+
+## Deprecation of the AzureRm.SiteRecovery module
+
+The AzureRm.SiteRecovery module will be deprecated in an upcoming release.
+
+The AzureRm.SiteRecovery module is being superseded by the AzureRm.RecoveryServices.SiteRecovery module.
+
+The AzureRm.RecoveryServices.SiteRecovery is a functional superset of the AzureRm.SiteRecovery module and includes a new set of equivalent cmdlets and easy to use aliases to help you manage your Azure Site Recovery deployments.
+
+|--------------------------------------------------------|---------------------------------------------------------------------------------------------|----------------------------------------------------|
```

Comment:
@viverm the table is getting parsed incorrectly in markdown - removing the first and last row of this table fixed the issue for me